packages/Disclosure/src/disclosure.scss

```diff
@@ -122,20 +123,58 @@
     font-size: var(--ids-disclosure-font-size);
     line-height: var(--ids-disclosure-header-line-height);
     padding: var(--ids-disclosure-header-padding);
+    position: relative;
     text-align: left;
     width: 100%;
 
+    &::before {
+        border: var(--ids-disclosure-border-width-focus) solid transparent;
+        border-radius: var(--ids-disclosure-border-radius);
+        content: '';
+        display: block;
+        inset: 0;
+        position: absolute;
+    }
+
+    &--focus-visible::before {
+        border-color: var(--ids-disclosure-border-color-focus);
+    }
+
+    &:not(&--focus-visible):hover::before {
+        border-color: var(--ids-disclosure-border-color-hover);
+    }
+
+    &[aria-expanded='true']::before {
+        border-bottom-left-radius: 0;
+        border-bottom-right-radius: 0;
+    }
+
     .ids-disclosure--low-contrast & {
+        background: none;
         color: var(--ids-disclosure-low-contrast-header-color);
         padding: var(--ids-disclosure-low-contrast-header-padding);
     }
 
+    .ids-disclosure--low-contrast &::before {
+        border-radius: var(--ids-disclosure-border-radius);
+    }
+
+    .ids-disclosure--low-contrast &:not(&--focus-visible):hover::before {
+        border-color: transparent;
+    }
+
     &:hover,
-    &:focus {
-        background-color: var(--ids-disclosure-header-background-hover)
+    &--focus-visible {
+        background-color: var(--ids-disclosure-header-background-hover);
+
+        .ids-disclosure--low-contrast & {
+            color: var(--ids-disclosure-low-contrast-header-color-hover);
+            background: none;
```

Comment:
this should be `background-color: none`

Reply:
No, just a pet peeve, as it could override other background properties and have unwanted side effects.

Reply:
alright, updated.